### PR TITLE
Circles onboarding task

### PR DESCRIPTION
**Pull request by ana0**

### CODE REVIEW

**Comment by ana0** [comment]
Addresses #9

File: safe_relay_service/relay/views_v2.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
                           SafeMultisigEstimateTxResponseSerializer,
                           SafeMultisigEstimateTxResponseV2Serializer)
 from .services.safe_creation_service import SafeCreationServiceProvider
-from .tasks import deploy_create2_safe_task
+from .tasks import circles_onoboarding
```
Comment (adzialocha) [inline safe_relay_service/relay/views_v2.py:26]: *ono*boarding :eyes:

Comment (ana0) [reply safe_relay_service/relay/views_v2.py:26]: o no

Comment (adzialocha) [reply safe_relay_service/relay/views_v2.py:26]: Ok. Legit

**Pull request closed without merging**